datacapturing/src/androidTest/java/de/cyface/datacapturing/backend/BackgroundServiceTest.java

```diff
@@ -170,7 +170,7 @@ public void testStartDataCapturingTwice() throws InterruptedException, TimeoutEx
             @Override
             public void run() {
                 PongReceiver isRunningChecker = new PongReceiver(context);
-                isRunningChecker.pongAndReceive(2, TimeUnit.SECONDS, testCallback);
```

Comment:
Solves a TODO from you as this method was named wrongly

Reply:
Ok. but method names should always be verb phrases. In this case maybe 'checkIsRunningAsync' or something along the lines.